### PR TITLE
chore: [POC] autocomplete with evalValues

### LINKED ISSUE #33747: [Task]: POC for getting the full path for an autocomplete suggestion sub property
### SubTasks

- [x] To get a full property path for a selected autocomplete suggestion : 
   Suppose we write `{{Api1.}}` in a text propery of a widget then an autocomplete suggestion box opens up i.e. `data`, `isLoading`, `responseMeta` and the tooltip for the first item is opened by default e.g. `data`, then we should be able to get the full path i.e. `Api1.data` in the tooltip wrapper using the codemirror tern service. Reference : https://www.notion.so/appsmith/Technical-Feasibility-6fae387590444b8184b989431f2c5419?pvs=4#081c05da3a084a33b37c15ba3ad4782d
  Screenshot 2024-05-27 at 12 48 07 AM

- [x] To be able to get the data from the evaluations in the tooltip wrapper container `ternDocTooltip.tsx` component file to display the evaluated values in the ui area.
  Screenshot 2024-05-27 at 12 54 01 AM

Figma file : https://www.figma.com/design/85kYou9qfqrxMXFuJLDKeU/Integrations-and-Templates%2FBuilding-blocks?node-id=1300-20047&t=iZMrHc5rTbkDEwmI-0

Related Notion docs : [Bindings and State inspector](https://www.notion.so/appsmith/Bindings-and-State-inspector-7616778b6a0a4dba8a85fc520c7e0beb) , [Technical Feasibility](https://www.notion.so/appsmith/Technical-Feasibility-6fae387590444b8184b989431f2c5419)

### PR DESCRIPTION
**Pull request by rishabhrathod01**
## Description

Fixes #33747 

## Automation

/ok-to-test tags=""

### :mag: Cypress test results

> [!CAUTION]  
> If you modify the content in this section, you are likely to disrupt the CI result for your PR.

## Communication
Should the DevRel and Marketing teams inform users about this change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: app/client/src/utils/autocomplete/CodemirrorTernService.ts [hunk 1]
```diff
@@ -404,6 +405,25 @@ class CodeMirrorTernService {
     this.server.deleteDefs(name);
   }
 
+  getParentPath(inputQuery: string) {
+    // inputQuery is "app" then parentPath will be ""
```
Comment (alex-golovanov) [inline app/client/src/utils/autocomplete/CodemirrorTernService.ts:409]: It may be better to write a unit test and convert these to test cases.

Comment (rishabhrathod01) [reply app/client/src/utils/autocomplete/CodemirrorTernService.ts:409]: @alex-golovanov Agreed. The test is skipped here as this POC PR. It won't be merged into the release branch.

**Comment by rishabhrathod01** [comment]
POC task is completed and the code here will be used in the next task by @sneha122

**Pull request closed without merging**